### PR TITLE
COL-1715, 'bookmarklet.js' is what is put to browser toolbar

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira-secure.berkeley.edu/browse/COL-1715

A work in progress.

### CODE REVIEW

**Comment by johncrossman** [comment]
@lyttam or @pauline2k  - when you have moment, review needed.

File: src/assets/bookmarklet/bookmarklet.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright ©2020. The Regents of the University of California (Regents). All Rights Reserved.
```
Comment (pauline2k) [inline src/assets/bookmarklet/bookmarklet.js]: Obsolete!!!

Comment (johncrossman) [reply src/assets/bookmarklet/bookmarklet.js]: Uh oh, gotta keep our copyright right. I'll amend.

**Comment by pauline2k** [comment]
Looks good to me, @johncrossman, unless we care about that copyright date.

**Comment by johncrossman** [comment]
@pauline2k - Amended.

**Pull request merged**